lib/fastlane/plugin/dynatrace/helper/dynatrace_helper.rb

```diff
@@ -114,6 +114,65 @@ def self.get_dss_client(params)
         return dtxDssClientPath
       end
 
+      def self.symlink_lldb(lldb_path, destination_path, auto_link = true)
+        require_path(destination_path)
+
+        unless lldb_path.nil?
```

Comment:
Shouldn't there be also an error message?

Reply:
actually this is an early-return, the error message would be at the end of the method depending on the other cases.